config/manufacturer_specific.xml

```diff
@@ -63,6 +64,10 @@
 		<Product type="0003" id="004b" name="Smart Energy Switch v2" config="aeon_labs/ses2.xml"/>
 		<Product type="0004" id="0025" name="Z-Wave Repeater"/>
 		<Product type="0004" id="0050" name="DSD31 Siren Gen5" config="aeon_labs/dsd31.xml"/>
+        <!-- US-market products start with type prefix "01" -->
```

Comment:
I added a comment here instead of putting it in the name, as the product country distinction is likely unimportant to the end-user who will be seeing the "name" value displayed.